### PR TITLE
Release 13.3.0

### PR DESCRIPTION
**Pull request by fheinecke**
This is my first release - would appreciate a more thorough review than normal (preferably not just a quick stamp)

### CODE REVIEW

File: CHANGELOG.md
Comment (tcsc) [inline CHANGELOG.md:5]:
```suggestion
* Added azure support to Helm charts[#29734](https://github.com/gravitational/teleport/pull/29734)
```

**Review by tcsc — APPROVED** [review]

File: CHANGELOG.md
Comment (tcsc) [inline CHANGELOG.md:8]: I'm in 2 minds about including dependency updates. On one hand our changelogs have been feature-oriented. But on the other, I can also see that someone watching for fixes would still be interested in this.

Comment (camscale) [reply CHANGELOG.md:8]: I excluded the buf update because that seemed to not matter as a rather dev-oriented thing. I did include the Go toolchain update because I noted it was previously in changelogs, perhaps because security conscious users may want to know we are not using a vulnerable version of the toolchain/libraries.

Comment (zmb3) [reply CHANGELOG.md:8]: 👍 Go or Rust updates make a lot of sense. I wouldn't include buf updates.

Comment (fheinecke) [reply CHANGELOG.md:8]: Thanks all. Comments make sense and I agree. I'll remove the line.

**Comment by fheinecke** [comment]
Holding off on merge (expected tomorrow) until some of @jakule 's PRs land

File: CHANGELOG.md
Comment (jakule) [inline CHANGELOG.md:3]: Another change that I'm backporting https://github.com/gravitational/teleport/pull/29754
> Add `max_duration` role option allowing to request role access up to 7 days

File: CHANGELOG.md
Comment (zmb3) [inline CHANGELOG.md:5]:
```suggestion
* Added Azure support to Helm charts [#29734](https://github.com/gravitational/teleport/pull/29734)
```

File: CHANGELOG.md
Comment (zmb3) [inline CHANGELOG.md:13]: I think we can remove this one.

**Review by zmb3 — COMMENTED** [review]
I would go back to grouping the changes like Roman did most recently in the 13.2.2 release.

I'd also try to emphasize Azure support a bit more. Since this is a new minor release and not just a list of bug fixes, I don't think you have to stick to an unordered list.

Maybe something like:

```
New backends

Teleport 13.3 includes a new Postgres backend that supports both
cluster state and the audit log. Additionally, Azure users can now
leverage Azure blob storage for session recordings.

<link to docs>
```

**Comment by fheinecke** [comment]
@zmb3 

> I would go back to grouping the changes like Roman did most recently in the 13.2.2 release.

I can go this route if you like but the work required for me to do so will likely delay the release another day, which also heavily cuts into the time I have to deploy and test parallel OS package publishing before the v12 and v11 releases.

I think the grouped changes approach is great for our customers, but with how we're currently handling changelogs (release manager handles 100% of this) it's pretty difficult to do. I took a look at the PRs, and out of the 14 PRs listed in this changelog I can guess what product/feature _maybe_ a third of the changes are associated with, which means that I need to contact and get a response from the owners of the other ten PRs. Again, I can do this, but it will take additional time to get this release (and the others this week) out the door.

> `<link to docs>`

Do you know where I can find our docs on this? I searched for "azure blob storage" on our docs and came up with nothing related to session recording (as far as I can tell).

**Comment by espadolini** [comment]
> Do you know where I can find our docs on this? I searched for "azure blob storage" on our docs and came up with nothing related to session recording (as far as I can tell).

in this specific case the docs are still on the way, but AFAICT we must wait for (or quite precisely synchronize) the release with the docs backport, because we shouldn't have main version docs for features that are not available yet.

Can we edit the changelog at a later time? Maybe we could add the docs link at a later time.

**Review by r0mant — APPROVED** [review]

**Pull request merged**
